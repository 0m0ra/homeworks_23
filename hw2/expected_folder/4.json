{
    "min age": 26,
    "max age": 55,
    "avg age": 39,
    "median age": 37,
    "< 2 days": 0.0,
    "< 1 week": 0.0,
    "< 1 month": 0.0,
    "< 6 months": 0.0,
    "> 6 months": 0.0
}