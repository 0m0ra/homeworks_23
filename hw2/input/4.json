{
    "user": {
        "region": "Omsk",
        "registered": "2019-10-15",
        "last_login": "2013-09-09",
        "email": "user@yandex.ru",
        "age": 61
    },
    "user2": {
        "region": "Tomsk",
        "registered": "1999-03-07",
        "last_login": "2008-07-22",
        "email": "user2@gmail.com",
        "age": 45
    }
}