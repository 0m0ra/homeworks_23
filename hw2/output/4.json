{
   "email": {
      "yandex.ru": 33.33,
      "gmail.com": 33.33,
      "mail.ru": 33.33
   },
   "registered": {
      "2012-12-24": 33.33,
      "2019-10-30": 33.33,
      "2022-10-30": 33.33
   }
}