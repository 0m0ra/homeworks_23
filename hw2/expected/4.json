{
   "0-18": 0.0,
   "18-25": 0.0,
   "26-45": 50.0,
   "45-59": 0.0,
   "60+": 50.0,
   "2019": 50.0,
   "1999": 50.0
}